pylmod/tests/test_gradebook.py

```diff
@@ -267,7 +345,31 @@ def test_get_gradebook_id(self):
         # Remove data and assert exception raised
         self._register_get_gradebook(False)
         with self.assertRaises(PyLmodUnexpectedData):
-            gradebook_id = test_gradebook.get_gradebook_id(self.GBUUID)
+            test_gradebook.get_gradebook_id(self.GBUUID)
+
+    @httpretty.activate
+    def test_get_options(self):
+        """Verify that we can get the options for a gradebook."""
+        test_body = {'data': {'membershipQualifier': '/project/mitxdemosite'}}
+        httpretty.register_uri(
+            httpretty.GET,
+            '{0}gradebook/options/{1}'.format(
+                self.GRADEBOOK_REGISTER_BASE,
+                self.GRADEBOOK_ID
+            ),
+            body=json.dumps(test_body)
+        )
+        self._register_get_gradebook()
+        gradebook = GradeBook(self.CERT, self.URLBASE, self.GBUUID)
+        options = gradebook.get_options(gradebook_id=self.GRADEBOOK_ID)
+        self.assertEqual(
+            options['membershipQualifier'],
+            test_body['data']['membershipQualifier']
+        )
+        # check for no data
+        self._register_get_options(False)
```

Comment:
It appears you are now doing it both ways, you are now using that URL twice, but you are registering for it in two different locations.  Additionally, you still don't need a method for this, just delete the `data` key from `test_body` and you won't get data back, no need for branches and a function.